Source/GUI/dvrescue/dvrescue/DefaultToolTip.qml

```diff
@@ -2,6 +2,11 @@ import QtQuick 2.0
 import QtQuick.Controls 2.12
 
 ToolTip {
-    delay: 1000
+    id: control
+    delay: 500

```

Comment:
and tried with 100 and liked that. Is there a separate variable to control the animation speed. Sometimes I'm just in a hurry to get my metadata. :)